Jenkinsfile

```diff
@@ -37,7 +37,7 @@ pipeline {
         stage('Build') {
             when {
                 expression {
-                    return env.shouldBuild == "true"
```

Comment:
`== "true"` didn't work?

Reply:
doesn't work for the very first build( with no tags) where pipeline params are not populated even when they have a default value associated with them. known Jenkins bug - https://issues.jenkins-ci.org/browse/JENKINS-40574